extensions/interactions/EndExploration/directives/oppia-interactive-end-exploration.component.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
Can you please use `git mv ....` command to rename the file and then do the changes on the renamed file? (This will help us keep the history of the file, will reduce the number of changes in the commit, will help you get quick reviewers from reviewers (with lessr change).)